src/OVAL/results/oval_cmp_impl.h

```diff
@@ -29,16 +29,6 @@
 #include "oval_system_characteristics.h"
 
 
-/**
- * Compare state entity (or variable/value) to sysent object collected from system.
- * This function does not support @datatype="record".
- * @param state_value Value defined within state/entity/value or variable/value
- * @param sysent Value collected from system
- * @operation Comparison type operation
- * @returns OVAL Result of comparison
- */
-oval_result_t oval_ent_cmp_str(char *state_data, oval_datatype_t state_data_type, struct oval_sysent *sysent, oval_operation_t operation);
-
```

Comment:
Shouldn't we keep this for API compatibility?

Reply:
Mmm, it looks like a private API.

Reply:
@jan-cerny Am I right?

Reply:
`abi-compliance-checker` is OK with this change.

Reply:
@evgenyz @yuumasato Yes, this is a private function. Public API functions have `OSCAP_API` decorator and they are usually located in `public` directories.